salt/modules/useradd.py

```diff
@@ -470,10 +473,10 @@ def chfullname(name, fullname):
 
         salt '*' user.chfullname foo "Foo Bar"
     '''
-    return _update_gecos(name, 'fullname', fullname)
+    return _update_gecos(name, 'fullname', fullname, root)
```

Comment:
Uhmm this is not correct. I realized that _update_gecos accept the root parameter, used to usermod, but this is deceptive as the return value use _get_gecos, that use pwd.getpwnam, that do not have a root parameter.

Maybe I can implement an alternative to pwd.* in case that root parameter is provided, and if not falback to pwd